builtin/mainmenu/settings/dlg_settings.lua

```diff
@@ -151,6 +151,72 @@ local function get_setting_info(name)
 end
 
 
+-- These must not be translated, as they need to show in the local
+-- language no matter the user's current language.
```

Comment:
I was wondering why some language do not have its local name written in the code. Is it because of RTL text direction? If that is true, we need to add a comment to explain the reasoning behind it. Additionally, we can still use the Latinised version of the local name in the code for now (see: `ms_Arab`):
- `ar`: Al-ʿArabiyyah
- `dv`: Dhivehi
- `he`: Ivrit